server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -726,7 +726,8 @@ public void apply(Settings value, Settings current, Settings previous) {
             TelemetrySettings.TRACER_SAMPLER_PROBABILITY,
             TelemetrySettings.METRICS_PUBLISH_INTERVAL_SETTING,
             TelemetrySettings.TRACER_FEATURE_ENABLED_SETTING,
-            TelemetrySettings.METRICS_FEATURE_ENABLED_SETTING
+            TelemetrySettings.METRICS_FEATURE_ENABLED_SETTING,
+            TelemetrySettings.TRACER_SAMPLER_ACTION_PROBABILITY
```

Comment:
Please remove this setting

Reply:
Added here with the intention that probability can be updated while OS is running